float64 lat	#rads
float64 lng	#rads
float64 alt	#meters
float64 vx	#meters/sec
float64 vy	#meters/sec
float64 vz	#meters/sec
float64 course  #rads
float64 q0
float64 q1
float64 q2
float64 q3
float64 angle_x	#rads
float64 angle_y	#rads
float64 angle_z #rads

uint8 satellites
uint8 quality
time time
uint8 flight_status

string deb
